index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
You need to add html files to a folder 'templates', currently it is not able to fetch html pages when executed.

Reply:
Ma'am, I have already created the Templates folder inside my PC. I have just uploaded them for review. 
Should I send the folder too ?

Reply:
In order for the code to be successfully executed on any machine, you need to make sure that the html files in a folder named 'templates' in the repository too.

Reply:
Okay got it.